docs/index.md

```diff
@@ -4289,11 +4289,42 @@ reverse chronological order. We'll emphasize user-visible changes.
 
 #### Version 16
 
-* [Goal labels are validated as strictly as hypothesis labels](https://github.com/expln/metamath-lamp/issues/81)
-* [Adding a "reset" option](https://github.com/expln/metamath-lamp/issues/21)
-* [Consistently determining replacement fields](https://github.com/expln/metamath-lamp/issues/82)
-* [Make it more obvious how to get a completed proof](https://github.com/expln/metamath-lamp/issues/11)
-* [Enable easily preloading an existing proof](https://github.com/expln/metamath-lamp/issues/8).
+* [Issue 8](https://github.com/expln/metamath-lamp/issues/8) Possibility to preload an existing proof (see more details below)
+* [Issue 77](https://github.com/expln/metamath-lamp/issues/77) Use the unification algorithm to find substitutions (see more details below)
+* [Issue 81](https://github.com/expln/metamath-lamp/issues/81) For goal labels, apply same strict label validation rules as for hypotheses
+* [Issue 21](https://github.com/expln/metamath-lamp/issues/21) Add a "reset" option
+* [Issue 82](https://github.com/expln/metamath-lamp/issues/82) Replacement fields should use selections consistently
+* [Issue 11](https://github.com/expln/metamath-lamp/issues/11) When the goal statement is proved then the completed proof is shown automatically.
+* [Issue 47](https://github.com/expln/metamath-lamp/issues/47) Automate merging of statements (see more details below)
+* [Issue 149](https://github.com/expln/metamath-lamp/issues/149) Allow merging multiple statements with one merge button click
+* The Ability to specify number of assertions per page on the
+  proof explorer tab.
+* The algorithm for creating proof tables was changed a bit, so now
+  it should produce same proof tables as on metamath.org. (If you
+  find a discrepancy between proofs tables on metamath.org and mm-lamp
+  for the same proof, then please feel free to let me know.)
```

Comment:
`please feel free to let me know` - I think this should be changed: `please feel free to file an issue in the mm-lamp repository - https://github.com/expln/metamath-lamp/issues`

Reply:
100% agree, that's much better. I'm fixing it right now.